invenio_communities/assets/semantic-ui/js/invenio_communities/community/featured-communities/FeaturedCommunities.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import { withCancel } from "react-invenio-forms";
+import { http } from "../../api/config";
+import { Grid, Message, Container } from "semantic-ui-react";
+import FeaturedCommunity from "./FeaturedCommunity";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_communities/i18next";
+
+export default class FeaturedCommunities extends Component {
```

Comment:
Isn't this duplicated from: https://github.com/inveniosoftware/invenio-communities/pull/836/ ?

Reply:
Yes, check PR message, only review last commit :wink: sorry for the confusion